### DISCUSSION TITLE
Manual run showing after schedule run independent of time

### DISCUSSION THREAD

**Discussion by vishalyadav93** [apache/airflow#31124] [Q&A]
### Apache Airflow version

2.3.3 (latest released)

### What happened

In Tree view of airflow, manual run is always showing after schedule run even though manual run is before schedule run

### What you think should happen instead

UI issue

### How to reproduce

You can run a dag one schedule_run other manual_run. The manual_run should be run before schedule_run , in Tree view of the airflow UI , the manual run will show after schedule run

### Operating System

linux

### Deployment

Docker-Compose

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by eladkal** [comment]
Airflow 2.3.3 doesn't have Tree view.
Can you clarify what are you referring to?

**Comment by potiuk** [comment]
I believe also the sorting has been fixed to follow "actual" run date in one of the later versions of airflow - you can look it up in the release notes/issues @vishalyadav93

**Comment by potiuk** [comment]
Yes. That was one of the highlighs of 2.4.0 : https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#dag-runs-sorting-logic-changed-in-grid-view-25090
